frontend/src/components/filter/FilterSection.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Typography } from '@material-ui/core';
+
+interface Props {
+  title: string;
+  children: React.ReactNode;
+}
+
+const FilterSection = ({ title, children }: Props) => {
+  return (
+    <>
+      <Typography
+        variant="h6"
+        style={{ borderBottom: '1px solid #E7E5DB', marginBottom: '0.3em' }}
```

Comment:
I would not recommend using inline style in our components.

The day we will add a Content Security Policy in the front end, inline style will be blocked if not explicity authorized. And authorizing inline style would be a security flaw.

See: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/style-src#unsafe_inline_styles

Reply:
I updated the PR to use `className` instead of inline `style`.  

Although your point about Content-Security-Policy is valid, enforcing such a policy everywhere would probably be constraining. Usage of inline styles is quite deeply baked into Material UI: see https://github.com/mui-org/material-ui/issues/19938

Reply:
You're right.

I'm curious about how many mui components use inline style.